scipy/special/basic.py

```diff
@@ -1484,7 +1497,20 @@ def bi_zeros(nt):
         Values of Bi(x) evaluated at first `nt` zeros of Bi'(x)
     bip : ndarray
         Values of Bi'(x) evaluated at first `nt` zeros of Bi(x)
-
+    
+    Examples
+    --------
+    >>> from scipy import special
+    >>> b, bp, bi, bip = special.ai_zeros(3)
```

Comment:
I think the copy paste didn't end well here 😃

Reply:
Oops. I didnt notice it. Let me check the values for `bi_zeros` again.